### DISCUSSION TITLE
Epic Games cannot join Dedicated Servers

### DISCUSSION THREAD

**Discussion by evilfactory** [FakeFishGames/Barotrauma#14553] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Dedicated servers don't appear in the server list, and if you try to direct join it just says "Authentication Failed".

### Reproduction steps

1. Start game on Epic Games
2. Look at the server list, no dedicated servers
3. Try direct joining via the IP Address a dedicated server

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Unfortunately there's no dedicated server available for Epic Online Services, and it's only possible to join SteamP2P servers using crossplay.

**Reply by evilfactory** [reply]
I swear I remember joining a dedicated server as epic games and it working fine when we were testing EOS on undertow, odd
